### ISSUE TITLE
Release without setup (Windows)

### ISSUE THREAD

**Issue by twoco** [4ian/GDevelop#4348]
Provide a Windows build without setup. Just a zip. Extract it, place it anywhere and run it.

**Comment by willianholtz** [comment]
But the .exe version can be extracted normally. I always do this so I don't have it installed!

**Comment by Silver-Streak** [comment]
For the Engine itself: The exe is  compressed installer. Just open it with 7zip and you can extract the app-64 zip from within it, which is the full application , no install needed.

For game exports: What you are describing is already in the engine.

When you choose "export to other platforms" using the external online build service, and select desktop, just choose the zip file as the export option.

Local builds give you an uncompressed folder that you can zip in addition to the installer.

**Comment by twoco** [comment]
Yes, I mean the setup of GDevelop like `GDevelop-5-Setup-5.0.144.exe` ... Ok, in this case I can extract the setup.exe with 7zip and extract the file `$PLUGINSDIR/app-64.7z` as well. It seems to work. I wonder what does the setup configure (registry, etc.)? Anyway, consider to provide the zip/7z file to download beside the setup, etc. Close if you won't. It's just a suggestion.

**Comment by 4ian** [comment]
For now I've added a page about this here: https://wiki.gdevelop.io/gdevelop5/getting_started/windows-portable-version-gdevelop, reachable from https://wiki.gdevelop.io/gdevelop5/getting_started - so that it can be found from a search engine. Feel free to adapt/add any comment that would be useful here!
